common/locales/en/challenge.json

```diff
@@ -31,6 +32,8 @@
     "challengeTitle": "Challenge Title",
     "challengeTag": "Tag Name",
     "challengeTagPop": "Challenges appear on tag-lists & task-tooltips. So while you'll want a descriptive title above, you'll also need a 'short name'. Eg, 'Lose 10 pounds in 3 months' might become '-10lb' (Click for more info).",
+    "moreInfoChallenges": "<a target='_blank' href='http://habitica.wikia.com/wiki/Challenges' class='pull-right'><small>More Info</small></a>",
+    "moreInfoTags": "<a target='_blank' href='http://habitica.wikia.com/wiki/Tags' popover='Challenges appear on tag-lists &amp; task-tooltips. So while you&#39;ll want a descriptive title above, you&#39;ll also need a &#39;short name&#39;. Eg, &#39;Lose 10 pounds in 3 months&#39; might become &#39;-10lb&#39; (Click for more info).' popover-trigger='mouseenter' popover-placement='right' class='hint vertical-20'>More Info</a>",
```

Comment:
This line has a lot of HTML tag attributes in it (`popover-trigger='mouseenter'` etc) which concerns me because a translator could mistakenly change them. It might be better to handle this in a similar way to `contribWikiaLink ` in `website/views/options/inventory/pets.jade`